src/server/task-list/controller.test.js

```diff
@@ -174,14 +179,15 @@ describe('#taskListController', () => {
     )
 
     expect(statusCode).toBe(statusCodes.ok)
-    expect(payload).toEqual(expect.stringContaining(`5 out of 5`))
+    expect(payload).toEqual(expect.stringContaining(`6 out of 6`))
     expect(payload).toEqual(expect.stringContaining('govuk-button--secondary'))
   })
 
   it('should state all section complete, and have a green button', async () => {
     await session.setState('origin', validOriginState)
     await session.setState('destination', validDestinationState)
     await session.setState('licence', validLicenceState)
+    await session.setState('identification', validIdentificationState)
```

Comment:
random fact
recently learnt that doing multiple `await`s like this is upto 8x slower than doing `await Promise.all([prom1, ...promN])`